homeassistant/components/xiaomi_miio/sensor.py

```diff
@@ -520,7 +520,7 @@ class XiaomiMiioSensorDescription(SensorEntityDescription):
         key=ATTR_CONSUMABLE_STATUS_MAIN_BRUSH_LEFT,
         parent_key=VacuumCoordinatorDataAttributes.consumable_status,
         name="Main Brush Left",
-        entity_registry_enabled_default=False,
+        entity_registry_enabled_default=True,
```

Comment:
That is the default, so we can just remove this line.

```suggestion
```

Reply:
Indeed, let's clean it up :+1: